CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# Unreleased
+
+Please use this place to add information of work that hasn't been released,
+and specify if that is backwards-compatible.
+
+# 30.6.0
+
+* Add grouped content and links
```

Comment:
The adapters for this haven't been merged yet (because we are going to change the endpoint)

The closed PR is https://github.com/alphagov/gds-api-adapters/pull/510